### ISSUE TITLE
Question : Direct Seeders

### ISSUE THREAD

**Issue by sootysax** [DeckerMMIV/FarmSim_Mod_SoilMod#5]
do direct seeding tools work as cultivators in regard to soil mod?

**Comment by DeckerMMIV** [comment]
That depends on how you perceive what a 'cultivator' would do, compared with what a 'sowing-machine using direct drilling/seeding' would do.

In this SoilMod, a tool with the `cultivator` or `plough` vehicle-type, will affect the foliage-layers for manure/slurry/lime/solid-fertilizer/weeds, due to the specializations (Cultivator.LUA & Plough.LUA) uses the `Utils.updateCultivatorArea()` and `Utils.updatePloughArea()` methods. I have specifically added a plugin-effect for just that.

However a `sowingMachine` vehicle-type (SowingMachine.LUA) uses the `Utils.updateSowingArea()` method, which I have not added any plugin-effect for regarding the manure/slurry/lime/solid-fertilizer. Only a plugin-effect to remove weeds.

The game's default effects, that these three vehicle-types normally cause on the ground, are still in effect. I.e. changes the fields ground-texture (cultivated/plowed/seeded) and places seeds.

**Comment by sootysax** [comment]
so in other words, a direct seeder will not effect lime, slurry or manure that is on the soil in the same way a cultivator would.   it will only remove weeds, and basically waste any layer of manure, lime or slurry on the field?

**Comment by DeckerMMIV** [comment]
A `sowingMachine` will (currently) not affect the manure/slurry/lime/fertilizer - so it will still be there and not exactly "wasted" (well, the manure will be.)

I have speculated about causing damage to a `sowingMachine`, if it is used on a field where there is visible manure. - However this "damage" would need to be a more general mod for all vehicles/tools, and not specific to SoilMod.

**Comment by sootysax** [comment]
what about the Zunhammer Zunidisc??  it is a cultivator that attaches to the back of the SKE 18.5 PU Slurry tanker..   in fs13, this would immediately cultivate the slurry into the ground..   

it seems as if cultivating slurry has no effect, only growth stage will effect it..   this kind of makes the Zunidisc not useful

**Comment by DeckerMMIV** [comment]
The way the Zunidisc "worked" in SoilMod for FS2013 was not intended. - It should never have been able to increase the organic-fertilizer in the way it did.

For SoilMod-FS15 I have "fixed" this problem, by only allowing slurry to increase the N+PK at growth-cycle. Though there is a "bug", where the (FS15) Zunidisc's cultivator-part isn't actually doing any cultivating. I would need to see the game's base scripts to get more insight in this. Hopefully GIANTS will publish the new ones I'm curious about; `WorkArea.LUA`.

**Comment by sootysax** [comment]
Fair enough - so is slurry is basically just another spray type - not as strong as Fertlizer N, but doesn't decrease pH values like N does.

So is slurry traditionally applied pre-seeding, post-plowing/cultivation?or should it be sprayed after the first growth stage (like the synthetic fertilizers)?

Not sure if there is still that restriction on synthetic fertilizers like in FS13 (must be sprayed after the first growth stage)

**Comment by DeckerMMIV** [comment]
There is no "organic-/synthetic-fertilizer" in SoilMod-FS15. Its just "fertilizer" for N+PK which can be applied at any time.
